doc/how_to_contribute/how_to_write_how_to_guides.rst

```diff
@@ -0,0 +1,47 @@
+How-To Write a How-To Guide
+===========================
+
+The How-To Guides are distinctly different from tutorials in that they are directed towards a different audience.
+You should assume much more knoledge of MoveIt and simply offer a quick guide for a specific problem.
+These pages posses these qualities:
+
+* problem oriented
+* a series of steps
+* a focus on the goal
+* addressing a specific question
+* no unnecessary explanation
+* a little flexibility
+* practical usability
+* good naming
+
+Not a Tutorial
+--------------
+
+In a tutorial you carefully guide the reader through doing a set steps with the goal of learning.
+Readers who are following tutorials don't know the terms or concepts yet to ask specific questions.
+Readers of How-To guides are already using MoveIt but are looking for isntructions on doing something specific.
+
+Names Answer a User's Question
+------------------------------
+
+How-To guies answer very specific questions and should capture Google searches.
+For that reason the naming of them is really important.
+An example of some potentially good names are:
+
+* How to Visualize Collisions in MoveIt
+* How to Grasp Objects with Moveit
+* How to run MoveIt with UR5
+* How to Fix a Segfault
+* How to Migrate from Foxy to Galactic
+* How to Run in Gazebo
+* How to Setup a New Robot for MoveIt
+* How to use the MoveIt RViz plugin
+* How to Teleop a Robot Arm with a Controller
+
+Goal Focused and Practical
+--------------------------
+
+For these guides it is really important that you focus specifically on solving the problem the user came to the site for.
+Because these users have a deeper understanding though you should present the user with some options when they will need to make a decision with tradeoffs.
+You should state what the assumed pre-requisites are in the introduction to the How-To Guide.
+That way you can avoid having multiple copies of isntructions on this site and allow users who already know the prerequisites to jump right into solving their problem.
```

Comment:
Could we make a template "How to Write a How-To Guide" that looks like what you want? Then folks will have a starting point for formatting.

Reply:
See my latest commit for this.